### PR TITLE
Enhance the .dockerignore man pages

### PR DESCRIPTION
**Pull request by rhatdan**
Use different examples then the reference page.

Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

#### What type of PR is this?

> /kind api-change
> /kind bug
> /kind cleanup
> /kind deprecation
> /kind design
> /kind documentation
> /kind failing-test 
> /kind feature
> /kind flake
> /kind other

#### Does this PR introduce a user-facing change?

```release-note

```

### CODE REVIEW

File: docs/buildah-add.md [hunk 1]
```diff
@@ -65,32 +65,32 @@ end with .go that are found in all directories.
...
 
-`**/temp*`
-Excludes files and directories starting with temp from any subdirectory.
+`**/output*`
+Excludes files and directories starting with `output` from any directory.
 
-`temp?`
-Excludes files and directories in the root directory whose names are a one-character extension of temp. For example, /tempa and /tempb are excluded.
+`src`
+Excludes files named src and the directorie src as well as all content in it.
```
Comment (TomSweeneyRedHat) [inline docs/buildah-add.md:81]: the any/all bit is a suggestion, your call, directorie needs to be changed though
```suggestion
Excludes files named src and the directory src as well as any content in it.
```

File: docs/buildah-add.md [hunk 2]
```diff
@@ -65,32 +65,32 @@ end with .go that are found in all directories.
...
-!README.md
+*.doc
+!Help.doc
 ```
 
-All markdown files except README.md are excluded from the context.
+Exclude all doc files except Help.doc from the image.
 
-For more information see:
+This functionality is compatible with the handling of .dockerignore files described here:
```
Comment (TomSweeneyRedHat) [inline docs/buildah-add.md:93]: :+1:

File: docs/buildah-bud.md [hunk 3]
```diff
@@ -739,32 +739,32 @@ end with .go that are found in all directories.
...
 
-`**/temp*`
-Excludes files and directories starting with temp from any subdirectory.
+`**/output*`
+Excludes files and directories starting with `output` from any directory.
 
-`temp?`
-Excludes files and directories in the root directory whose names are a one-character extension of temp. For example, /tempa and /tempb are excluded.
+`src`
+Excludes files named src and the directorie src as well as all content in it.
```
Comment (TomSweeneyRedHat) [inline docs/buildah-bud.md:755]:
```suggestion
Excludes files named src and the directory src as well as any content in it.
```

File: docs/buildah-copy.md [hunk 4]
```diff
@@ -63,32 +63,32 @@ end with .go that are found in all directories.
...
 
-`**/temp*`
-Excludes files and directories starting with temp from any subdirectory.
+`**/output*`
+Excludes files and directories starting with `output` from any directory.
 
-`temp?`
-Excludes files and directories in the root directory whose names are a one-character extension of temp. For example, /tempa and /tempb are excluded.
+`src`
+Excludes files named src and the directorie src as well as all content in it.
```
Comment (TomSweeneyRedHat) [inline docs/buildah-copy.md:79]:
```suggestion
Excludes files named src and the directory src as well as any content in it.
```

**Comment by rhatdan** [comment]
@vrothberg @nalind @ashley-cui @QiWang19 PTAL

**Pull request merged**